archive/cpio_old_le.ksy

```diff
@@ -0,0 +1,64 @@
+meta:
+  id: cpio_old_le
+  title: cpio archive, old binary variant, little-endian
+  file-extension: cpio
+  license: CC0-1.0
+  endian: le
+  encoding: ASCII

```

Comment:
Maybe worth adding pathNameAscii / pathNameUTF8 instances which makes the convertation as a property? This way maybe it will be even more easier for the user the understand what happens here.